definitions/divorce/json/CaseTypeTab/CaseTypeTab-resp-journey-roles-and-permissions-nonprod.json

```diff
@@ -0,0 +1,132 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "confidentialPetitionerForPetSol",
+    "TabLabel": "Confidential Petitioner",
+    "TabDisplayOrder": 13,
+    "CaseFieldID": "D8DerivedPetitionerCorrespondenceAddr",
+    "UserRole": "[PETSOLICITOR]",
+    "TabFieldDisplayOrder": 1,
+    "TabShowCondition": "D8PetitionerContactDetailsConfidential=\"keep\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "confidentialPetitionerForPetSol",
+    "TabLabel": "Confidential Petitioner",
+    "TabDisplayOrder": 13,
+    "CaseFieldID": "D8PetitionerPhoneNumber",
+    "TabFieldDisplayOrder": 2
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "confidentialPetitionerForPetSol",
+    "TabLabel": "Confidential Petitioner",
+    "TabDisplayOrder": 13,
+    "CaseFieldID": "D8PetitionerEmail",
+    "TabFieldDisplayOrder": 3
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "confidentialPetitionerForPetSol",
+    "TabLabel": "Confidential Petitioner",
+    "TabDisplayOrder": 13,
+    "CaseFieldID": "D8DerivedPetitionerHomeAddress",
+    "TabFieldDisplayOrder": 4
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "notesCitizen",
+    "TabLabel": "Notes",
+    "TabDisplayOrder": 20,
+    "CaseFieldID": "CaseNotes",
+    "UserRole": "citizen",
+    "TabFieldDisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "notesCourtAdmin",
```

Comment:
What's the difference between "notesCitizen" and "notesCourtAdmin"?

Reply:
Just the user role that can see it.